packages/transport-webrtc/test/stream.spec.ts

```diff
@@ -23,7 +23,8 @@ const MAX_MESSAGE_SIZE = 16 * 1024
 describe('Max message size', () => {
   it(`sends messages smaller or equal to ${MAX_MESSAGE_SIZE} bytes in one`, async () => {
     const sent: Uint8ArrayList = new Uint8ArrayList()
-    const data = new Uint8Array(MAX_MESSAGE_SIZE - 5)
+    const PROTOBUF_OVERHEAD = 5
```

Comment:
Maybe this should be exported from `src/stream.ts`, then it wouldn't need to be duplicated here?